workflow/UX-and-design-proposals.md

```diff
@@ -0,0 +1,147 @@
+---
+title: UX/design proposals, sharing process & feedback convention
+description: How to share UX & design proposals and guidelines on feedback convention
+---
+
+This document outlines the guidelines on our tools (e.g. Miro, Zeplin, GitHub) across the different disciplines and the different phases of a project. 
+
+## Table of content
+
+- [Feedback convention (across tools)](#feedback-convention)
+- [What tools do we use?](#what-tools-do-we-use)
+  - [Miro](#miro)
+  - [Docs & Specs](#docs-and-specs)
+  - [Zeplin](#zeplin)
+  - [General direction](#general-direction)
+- [Zeplin/Miro status convention](#zeplin-miro-status-convention)
+  - [Status and tags](#status-and-tags)
+  - [Projects name](#projects-name)
+  - [Sections name](#sections-name)
+  - [Links to specs/docs/epics](#links-to-specs-docs-epics)
+- [Who is it for?](#who-is-it-for)
+- [Inviting people to projects](#inviting-people-to-projects)
+- [When do we comment?](#when-do-we-comment)
+  - [Directions](#directions)
+  - [Does it need a comment?](#does-it-need-a-comment)
+- [Resolving comments](#resolving-comments)
+
+
+## feedback-convention
+
+Problems: semantic gap, hard to categorize the feedback,  need a way to receive feedback that can help us redesign the wireframes better and understand the motivation behind the feedback.
+ 
+Start your comment with:
+
+**1. I like… (tag a green colour on the Zeplin comment)**
+For components that you like and that we should keep it.
+ 
+**2. I wish… (tag a yellow colour on the Zeplin comment)**
+For components that you think
+- Can be improved
+- Disagree with and would like to propose an alternative behaviour
+- Would like to patternized
+- Need clarification (express which part is confusing)
+- If you have a question, please formulate it as "I wish...", this will be really helpful to learn about what you have in mind.
+ 
+**3. INFO: (tag a purple colour on the Zeplin comment)**
+- When you don’t want to express an opinion, but would like to share information, caveat, or references that I might have missed or overlooked in the prototype/wireframe.
```

Comment:
- When you don’t want to express an opinion, but would like to share information, caveat, or references that might have been missed or overlooked in the prototype/wireframe.